OpenRA.Game/Primitives/BitSet.cs

```diff
@@ -41,6 +41,22 @@ public static BitSetIndex GetBits(string[] values)
 			return bits;
 		}
 
+		public static BitSetIndex GetBitsNoAlloc(string[] values)
+		{
+			// Map strings to existing bits; do not allocate missing values new bits
+			BitSetIndex bits = 0;
+
+			lock (Bits)
```

Comment:
Minor style nit: `lock` should also use braces as the inner block is using them.

Reply:
Done.